src/Demo/main.c

```diff
@@ -0,0 +1,182 @@
+/**@file
```

Comment:
This application is for integration testing? Wouldn't it be better to include it in "test" directory, and name accordingly?

Reply:
Or, if this is not a test, but just a demo indeed, maybe it should be put in the top level directory?